ROADMAP.md

```diff
@@ -15,23 +15,21 @@ Please open an issue to track any initiative on the roadmap of Contour (Usually
-|Operational|Contour with managed Envoys|2021 H2||As part of the ongoing effort to support Gateway API, Contour will be managing the Envoy instances in a Contour deployment and existing ConfigMap representing the configuration will be redesigned as a CRD |
-|Deployment|Contour Operator GA|2021 H2|[projectcontour/contour-operator#205](https://github.com/projectcontour/contour-operator/issues/205)|Bring the Operator to Beta release and then to GA release suitable for production environments. The Operator will fully support Contour leveraging Gateway API|
-|General|Self Service Capabilities in Contour|2021 H2|[projectcontour/contour#2206](https://github.com/projectcontour/contour/issues/2206) mainly, with some others|It seems likely that some of these requests will be introduced by the Gateway APIs.|
 |Infrastructure|Incorporate Envoy go-control-plane to modernize xDS|2021 H2|[projectcontour/contour#2134](https://github.com/projectcontour/contour/issues/2134)||
+|General|Self Service Capabilities in Contour|2022 H1|[projectcontour/contour#2206](https://github.com/projectcontour/contour/issues/2206) mainly, with some others|It seems likely that some of these requests will be introduced by the Gateway APIs.|
+|Operational|Contour with managed Envoys|2022 H1||Contour will automatically manage the Envoy instances in a Contour installation, configured using the new Configuration CRD. Aside from being a useful change in general, this will also allow Contour to move more towards a fully dynamic Gateway API implementation.|
+|Deployment|Contour Operator GA|2022 H2|[projectcontour/contour-operator#205](https://github.com/projectcontour/contour-operator/issues/205)|Bring the Operator to Beta release and then to GA release suitable for production environments. This can't be completed until the managed Envoy work is complete. The Operator will fully support Contour leveraging Gateway API.|
+|Observability|Tracing support|2022 H1|[projectcontour/contour#399](https://github.com/projectcontour/contour/issues/399)|This triggered a deeper look at our existing ExtensionService and some possible changes might be needed|
 |Operational|Harden contour-authserver to provide a simple auth option|Unknown|[projectcontour/contour-authserver#14](https://github.com/projectcontour/contour-authserver/issues/14)|Contour-authserver is a small utility that translates between Envoy ext_auth gRPC and other protocols, this is to make it no longer experimental|
+|Observability|Support Access Log Service|Unknown|[projectcontour/contour#1691](https://github.com/projectcontour/contour/issues/1691)||
 |Security|Add OIDC support to contour-authserver|Unknown|[projectcontour/contour-authserver#13](https://github.com/projectcontour/contour-authserver/pull/13)||
```

Comment:
Should this be updated to reflect https://github.com/projectcontour/contour/issues/2664#issuecomment-958424324 where Envoy OAuth2 filter would be leveraged instead of contour-authserver?

Reply:
Yes, it should, thanks.